docusaurus/docs/e2e-test-a-plugin/locating-elements.md

```diff
@@ -0,0 +1,43 @@
+---
+id: locating-elements
+title: Locating elements for E2E tests
+description: How to locate elements on the page
+draft: true
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - e2e
+  - example test
+sidebar_position: 30
+---
+
+# Locating elements for E2E tests
+
+Ever since Grafana 7.0.0, end-to-end tests in Grafana have relied on selectors defined in the [`@grafana/e2e-selectors`](https://github.com/grafana/grafana/tree/main/packages/grafana-e2e-selectors) package to locate elements in the Grafana UI. 
+
+Locating Grafana UI elements can be challenging because the selector may be defined on either the `aria-label` attribute or the `data-testid` attribute. In the beginning, the selectors used the `aria-label` attribute, but now most selectors have been migrated to use the `data-testid` attribute instead.
+
+# Locating Grafana UI elements
+
+All pages defined by `@grafana/plugin-e2e` expose a `getByTestIdOrAriaLabel` method that locates elements using the right HTML attribute. Whenever you want to resolve a Playwright locator based on a Grafana UI selector, you should always use this method.
+
+```tsx
```

Comment:
I believe Docusaurus supports 'tsx' for typescript. See https://github.com/FormidableLabs/prism-react-renderer/blob/c914fdea48625ba59c8022174bb3df1ed802ce4d/packages/generate-prism-languages/index.ts#L9-L23

Reply:
This is actually not tsx. tsx should be used in typescript based react applications that uses jsx for rendering.